### ISSUE TITLE
Concatenate two times train_set 

### ISSUE THREAD

**Issue by Elpidam** [skaae/recurrent-spatial-transformer-code#4]
Hello! Recently I noticed that in `create_mnist_dataset.py`, in line 24:
`all_digits = np.concatenate([mnist_data['X_train'],
                             mnist_data['X_valid'],
                             mnist_data['X_train']], axis=0)`

the last part of the concatenation  is train set as it was the first part. This might be a mistake. Right?

**Comment by skaae** [comment]
Yes i think you are correct. I don't think it mattes because i do only use that all_digits for generating a list of distortions.

I corrected the error. Thanks for pointing it out :)
